packages/frameworks/src/frameworks.ts

```diff
@@ -1932,6 +1932,37 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'Storybook',
+    slug: 'storybook',
+    logo: 'https://api-frameworks.vercel.sh/framework-logos/storybook.svg',
+    tagline: 'Frontend workshop for UI development',
+    description:
+      'Storybook is a frontend workshop for building UI components and pages in isolation.',
+    website: 'https://storybook.js.org',
+    ignoreRuntimes: ['@vercel/next', '@vercel/node'],
+    detectors: {
+      every: [
+        {
+          matchPackage: 'storybook/.+',
```

Comment:
```suggestion
          matchPackage: 'storybook',
```

Reply:
This is not correct. The `storybook` package is a wrapper for `@storybook/cli`. Users don't install the `storybook` package as a dependency. Instead they run `pnpm dlx storybook@latest init` which added project specific packages like react vs preact vs vue.

Reply:
@cb1kenobi So in that case, it should check for `@storybook/cli`

Reply:
OK, not sure what's up with my example, but I don't have `storybook` in the `package.json`, but it's in `node_modules`. I created a new app and now `storybook` is in the `package.json`. Weirdness. PR updated to detect `storybook`. Good catch.